app/queries/display_ads/filtered_ads_query.rb

```diff
@@ -24,6 +24,8 @@ def call
       @filtered_display_ads = approved_and_published_ads
       @filtered_display_ads = placement_area_ads
 
+      return @filtered_display_ads if @area == "home_hero"
```

Comment:
I'm wondering do we need to do this early return or could we just let it go through the process, are there any functions that we're hoping to avoid further down? 

When we return early we don't get the benefit of the ordering by success rate, we also loose the other targeting of ads logged in and logged out.

Reply:
Thanks @lboogie04, here's my thoughts on those conditions: 

> These types of billboards are not able to target logged_in users for now, but we don't need to codify this as a validation.
I stand to be corrected but I think we're talking about targeting through the audience segments here. However, it says we don't need to validate this right now, so as long as the relationship is not created between the `display_ad` and the `audience_segment`, we should be okay.

> hero billboards cannot target tags...
As long as we don't pass `article_tags` in the `for_display` query, it will not target any tags.